pkgs/development/misc/vencord/default.nix

```diff
@@ -48,13 +49,13 @@ buildNpmPackage rec {
   ];
 
   installPhase = ''
-    cp -r dist/extension-unpacked $out
```

Comment:
maybe change it to:
`cp -r dist/ $out` for build
`cp -r dist/extension-unpacked $out` for buildWeb

This way only the necessary files for vencord web extension will be installed and there will be no need for the `/extension-unpacked` part in
`vencord = "${vencord-web-extension}/extension-unpacked";` (`webcord-vencord/default.nix`)